### PR TITLE
Fix supervisor-proc-exit-listener false alert during warm reboot issue.

### PR DESCRIPTION
**Pull request by liuh-80**
Fix supervisor-proc-exit-listener false alert during warm reboot issue: https://github.com/sonic-net/sonic-buildimage/issues/16686

#### Why I did it
supervisor-proc-exit-listener will generate false alert during warm reboot.

##### Work item tracking
- Microsoft ADO: 25295846

#### How I did it
Ignore alert message during warm reboot.

#### How to verify it
Pass all UT.
Manually verify issue fixed:

Sep 28 07:53:20.285652 vlab-01 ERR swss#supervisor-proc-exit-listener: message repeated 27 times: [ Process 'orchagent' is stuck in namespace 'host' (7.0 minutes).]
Sep 28 07:53:20.287377 vlab-01 INFO swss#supervisor-proc-exit-listener: Warm rebooting, Process 'orchagent' is stuck in namespace 'host' (7.0 minutes).

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [] 202205
- [] 202211
- [] 202305

#### Tested branch (Please provide the tested image version)

- [x]  master-16742.373711-493d8b7bf

#### Description for the changelog
Fix supervisor-proc-exit-listener false alert during warm reboot issue.

### CODE REVIEW

File: files/scripts/supervisor-proc-exit-listener [hunk 1]
```diff
@@ -86,8 +101,16 @@ def generate_alerting_message(process_name, status, dead_minutes):
...
         namespace = namespace_prefix + namespace_id
 
-    syslog.syslog(syslog.LOG_ERR, "Process '{}' is {} in namespace '{}' ({} minutes)."
-                  .format(process_name, status, namespace, dead_minutes))
+    message = "Process '{}' is {} in namespace '{}' ({} minutes).".format(
+                                                                    process_name,
+                                                                    status,
+                                                                    namespace,
+                                                                    dead_minutes)
+    if is_warm_reboot():
```
Comment (qiluo-msft) [inline files/scripts/supervisor-proc-exit-listener:109]: >is_warm_reboot [](http://example.com/codeflow?start=7&length=14)

Could this condition be more conservative? like is_warm_reboot and the orchagent is stuck at the same time?

Comment (liuh-80) [reply files/scripts/supervisor-proc-exit-listener]: Fixed, only ignore error log for orchagent stuck during warm reboot.

Comment (qiluo-msft) [reply files/scripts/supervisor-proc-exit-listener]: Seems like `WARM_RESTART_ENABLE_TABLE` is the indicator for warm-boot-up, not warm-shutdown which is what you want. I am considering orchagent process has the full visibility of freezing status, how about continuing hearbeat even when it is frozen?

Sorry for misleading.

Comment (liuh-80) [reply files/scripts/supervisor-proc-exit-listener]: Currently orchagent freeze by sleep, I will check if can improve the restart code in orchagent by stop handle all task:

                    SWSS_LOG_WARN("Orchagent is frozen for warm restart!");
                    sleep(UINT_MAX);

**Comment by liuh-80** [comment]
Close this PR, will fix the issue in Orchagent: https://github.com/sonic-net/sonic-swss/pull/2923

**Pull request closed without merging**